be/src/util/thrift_util.cpp

```diff
@@ -149,4 +149,24 @@ bool t_network_address_comparator(const TNetworkAddress& a, const TNetworkAddres
 
     return false;
 }
+
+void thrift_from_json_string(::apache::thrift::TBase* base, const std::string& json_val) {
```

Comment:
`set_thrift_from_json_string()` ?

Reply:
It's symmetric to `thrfit_to_json_string`. Why have to empahsize `set_`?